src/discussions/posts/post/PostLink.jsx

```diff
@@ -32,9 +36,10 @@ function PostLink({
           <span className="text-gray-700 x-small">{intl.formatMessage(messages.contentReported)}</span>
         </div>
       )}
-      <div className={`d-flex flex-row p-2 ${post.read ? 'bg-light-200' : ''}`}>
-        {!post.read && <Icon className="text-brand-500" src={Unread} />}
-        <Post post={post} />
+      <div className={`d-flex flex-row flex-fill mw-100 ${post.read ? 'bg-light-200' : ''}`}>
```

Comment:
+ please use [classnames](https://www.npmjs.com/package/classnames) for setting className conditionally.